ArrayUtils.js

```diff
@@ -1,21 +1,84 @@
 class ArrayUtils{
 
-    isEmpty(array){return false;}
 
-    append(original, value){return original;}
+     isEmpty(array){
+        if (array.length !== 0){
+        return false;
+        }
+    }
+    
+    append(original, value){
+        for (let i = 0; i <= original.length; i ++){
+            if(i === original.length -1){
+                original += value
+            }
+        }
+
+        return original
+    }
 
-    clone(original){return original;}
+    clone(original){
+        const array = []
+        for(let i = 0; i <= original.length; i++){
+            array += orginal
+        }
+        return array
+    }
 
-    subArray(original, from, to){return original;}
```

Comment:
Does not compile, `from` and `to` are indexed as arrays. `from` and `to` are numbers that represent a start and stop indices. (-11)